tests/runner/test/fixtures/async.test.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, expect, it } from '@rstest/core';
+
+describe('Test Async Suite', () => {
+  describe('0', async () => {
+    await new Promise<void>((resolve) => {
```

Comment:
Could move the sleep function to utils, we already have 4 similar functions (search `await new Promise`).